src/resolvers/mutations/users.js

```diff
@@ -74,6 +74,17 @@ const signup = async (_, { input }, context, _info) => {
             return utils.sendErrorResponse(field, message);
         }
 
+        if (input['department']) {
+            message = 'Department does not exist';
```

Comment:
Are we changing how we adding users?

Reply:
No in the client it will remain as before